docs/data/system/getting-started/usage/usage.md

```diff
@@ -202,7 +202,7 @@ const Div = styled('div')``;
 
 ### Any element with the babel plugin
 
-TODO [#23220](https://github.com/mui/material-ui/issues/23220).
+Visit [the to-do GitHub issue](https://github.com/mui/material-ui/issues/23220) about it.
```

Comment:
This would be amazing https://github.com/mui/material-ui/discussions/40844#discussioncomment-8422680

Reply:
I don't want to have this thread here necessarily, but I'd love to understand more about the rationale for getting rid of the Box. I can't understand it immediately.